docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -554,9 +554,99 @@ Payload objects with the following scopes take the form that follows:
 
 Consult the [pages section of the Channel Webhooks Guide](/docs/integrations/webhooks/events/channels#pages).
 
+## Price Lists
+The following price lists webhook events fire in response to actions that affect a specific price list:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceList/created`  | Fires when a price list is created. | 
+| `store/priceList/updated/` | Fires when a price list is updated. | 
+| `store/priceList/activated/` | Fires when a price list is activated. | 
+| `store/priceList/deactivated/` | Fires when a price list is deactivated. | 
+| `store/priceList/deleted/` | Fires when a price list is deleted. | 
```

Comment:
There is a `store/priceLists/deleted` batch event with a data payload like ```{"price_list_ids":[234]}```

Reply:
I added the schema for batch delete (`store/priceLists/deleted`) although it is confusing because I could only remove one price list at a time. I tried adding values separated by a comma but that did not work. 

If you have an example of this I can add the example to the endpoint. Also I see the `stores/priceList/deleted`  does not exist.
![Screenshot 2024-03-07 at 10 33 25 AM](https://github.com/bigcommerce/docs/assets/64287776/67e34911-165d-4ca6-a12a-952fac4b080e)